strax/chunk.py

```diff
@@ -206,9 +206,9 @@ def split(self, t: ty.Union[int, None], allow_early_split=False):
         """
         t = max(min(t, self.end), self.start)  # type: ignore
         if t == self.end:
-            data1, data2 = self.data, self.data[:0]
+            data1, data2 = self.data, self.data[:0].copy()
```

Comment:
This is to prevent the reference build between `data2` and `self.data` to make sure that the `sys.getrefcount(data)` is low enough.